### PR TITLE
Feat/find command

### LINKED ISSUE #17: 1. As a busy house-husband, I want to easily find out if I have a certain item in the fridge automatically so that I spend less time checking if I can cook certain meals.
Essentially a search function to check if user's item is in the fridge.

### PR DESCRIPTION
**Pull request by alvynben**
This pull request attempts to:

- Add a "find" function for users to search for ingredients by search term (Fixes #17)
- Simplifies the way Fridget prints separatorLines

### CODE REVIEW

File: src/main/java/seedu/ui/Ui.java [hunk 1]
```diff
@@ -80,7 +86,35 @@ public String getCurrentUserInput() {
...
+    }
+
+    /**
+     * Prints a list of ingredients, indented by four spaces.
+     * @param listOfIngredients The list of ingredients to be printed.
+     */
+    public void printListOfIngredients(ArrayList<Ingredient> listOfIngredients) {
+        int listSize = listOfIngredients.size();
+        for (Ingredient ingredient : listOfIngredients) {
+            printLine(FOUR_SPACE_INDENTATION + ingredient);
```
Comment (zonglun99) [inline src/main/java/seedu/ui/Ui.java:100]: Maybe can consider printing numbers before each ingredient too

File: src/main/java/seedu/commands/FindCommand.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import seedu.data.ingredient.Ingredient;
+import seedu.parser.Parser;
+import seedu.storage.IngredientList;
+import seedu.ui.Ui;
+
+import java.util.ArrayList;
+
+public class FindCommand extends Command {
+
```
Comment (zonglun99) [inline src/main/java/seedu/commands/FindCommand.java:11]: can delete this space here

File: src/main/java/seedu/ui/Ui.java (hunk 1 above)
Comment (alvynben) [reply src/main/java/seedu/ui/Ui.java:100]: Edited the function to allow both index and no index, so it is reusable.

**Review by zonglun99 — APPROVED** [review]
LGTM

**Pull request merged**